### ISSUE TITLE
Fourth controller gets all of them disconnected

### ISSUE THREAD

**Issue by Fusseldieb** [Ryochan7/DS4Windows#1863]
**Describe the bug**
Playing with 2 DS4 controllers work just fine, even 3, but as soon as I connect a fourth one, after some minutes of playing **all of them** become unresponsive and disconnect at the same time. 
I don't know if it is DS4 or the driver itself... I couldn't find anything related on the internet so far.

**To Reproduce**
Connect 4 DS4 controllers and play with them. After 5 minutes or so all of them become unresponsive and disconnect.

**Expected behavior**
Well... It shouldn't disconnect.

**Screenshots**
\-

**Desktop (please complete the following information):**
 - Controller Make and Model: Sony DS4 v.2 (4x CUH-ZCT2U)
 - Intel(R) Wireless Bluetooth(R) 20.30.2.1 (USB\VID_8087&PID_0AAA&REV_0002)
 - OS: Windows 10 Pro Build 2004
 - DS4Windows Version 2.2.5

**Additional context**
```
02/10/2021 22:52:49: DS4Windows version 2.2.5
02/10/2021 22:52:49: DS4Windows Assembly Architecture: x64
02/10/2021 22:52:49: OS Version: Microsoft Windows NT 10.0.19041.0
02/10/2021 22:52:49: OS Product Name: Windows 10 Pro
02/10/2021 22:52:49: OS Release ID: 2004
02/10/2021 22:52:49: System Architecture: x64
02/10/2021 22:52:49: Running as User
02/10/2021 22:52:50: Starting...
02/10/2021 22:52:50: Connection to ViGEmBus 1.16.112.0 established
02/10/2021 22:52:50: Searching for controllers...
02/10/2021 22:52:50: Using Shared Mode
02/10/2021 22:52:58: The gamepad DS4 v.2 (BT) uses custom feature set (MonitorAudio)
02/10/2021 22:52:58: Found Controller: A4:AE:11:F2:BC:C0 (BT) (DS4 v.2)
02/10/2021 22:52:58: Plugging in virtual X360 Controller
02/10/2021 22:52:58: Associate X360 Controller for input DS4 #1
02/10/2021 22:52:58: Controller 1 is using Profile "Default". Battery level is 100%
02/10/2021 22:53:03: The gamepad DS4 v.2 (BT) uses custom feature set (MonitorAudio)
02/10/2021 22:53:03: Found Controller: 28:C1:3C:7B:06:3C (BT) (DS4 v.2)
02/10/2021 22:53:03: Plugging in virtual X360 Controller
02/10/2021 22:53:03: Associate X360 Controller for input DS4 #2
02/10/2021 22:53:03: Controller 2 is using Profile "Default". Battery level is 100%
02/10/2021 22:53:10: The gamepad DS4 v.2 (BT) uses custom feature set (MonitorAudio)
02/10/2021 22:53:10: Found Controller: A0:AB:51:3E:01:BF (BT) (DS4 v.2)
02/10/2021 22:53:10: Plugging in virtual X360 Controller
02/10/2021 22:53:10: Associate X360 Controller for input DS4 #3
02/10/2021 22:53:10: Controller 3 is using Profile "Default". Battery level is 100%
02/10/2021 22:53:18: The gamepad DS4 v.2 (BT) uses custom feature set (MonitorAudio)
02/10/2021 22:53:18: Found Controller: 1C:A0:B8:90:67:4E (BT) (DS4 v.2)
02/10/2021 22:53:18: Plugging in virtual X360 Controller
02/10/2021 22:53:18: Associate X360 Controller for input DS4 #4
02/10/2021 22:53:18: Controller 4 is using Profile "Default". Battery level is 50%
02/10/2021 22:53:23: Controller 4's latency over 10ms (10.15 ms)
02/10/2021 22:53:23: Controller 2's latency over 10ms (10.35 ms)
02/10/2021 22:53:23: Controller 1's latency over 10ms (10.25 ms)
02/10/2021 22:53:23: Controller 3's latency over 10ms (10.05 ms)
02/10/2021 22:53:23: Controller 3's latency now under 10ms
02/10/2021 22:53:23: Controller 3's latency over 10ms (12.1 ms)
02/10/2021 22:53:24: Controller 3's latency now under 10ms
02/10/2021 22:53:24: Controller 2's latency now under 10ms
02/10/2021 22:53:24: Controller 3's latency over 10ms (11.65 ms)
02/10/2021 22:53:24: Controller 2's latency over 10ms (12 ms)
02/10/2021 22:53:24: Controller 1's latency now under 10ms
02/10/2021 22:53:24: Controller 1's latency over 10ms (10 ms)
02/10/2021 22:53:24: Controller 3's latency now under 10ms
02/10/2021 22:53:24: Controller 1's latency now under 10ms
02/10/2021 22:53:24: Controller 1's latency over 10ms (11.75 ms)
02/10/2021 22:53:24: Controller 3's latency over 10ms (11.85 ms)
02/10/2021 22:53:24: Controller 1's latency now under 10ms
02/10/2021 22:53:24: Controller 1's latency over 10ms (10 ms)
02/10/2021 22:53:24: Controller 1's latency now under 10ms
02/10/2021 22:53:24: Controller 4's latency now under 10ms
02/10/2021 22:53:24: Controller 2's latency now under 10ms
02/10/2021 22:53:24: Controller 3's latency now under 10ms
02/10/2021 22:53:31: Controller 1's latency over 10ms (11.35 ms)
02/10/2021 22:53:31: Controller 3's latency over 10ms (11.55 ms)
02/10/2021 22:53:31: Controller 2's latency over 10ms (10.85 ms)
02/10/2021 22:53:31: Controller 4's latency over 10ms (10.75 ms)
02/10/2021 22:53:31: Controller 4's latency now under 10ms
02/10/2021 22:53:31: Controller 4's latency over 10ms (11.85 ms)
02/10/2021 22:53:31: Controller 3's latency now under 10ms
02/10/2021 22:53:31: Controller 1's latency now under 10ms
02/10/2021 22:53:31: Controller 1's latency over 10ms (12.05 ms)
02/10/2021 22:53:31: Controller 3's latency over 10ms (11.55 ms)
02/10/2021 22:53:31: Controller 1's latency now under 10ms
02/10/2021 22:53:31: Controller 3's latency now under 10ms
02/10/2021 22:53:31: Controller 3's latency over 10ms (10.1 ms)
02/10/2021 22:53:31: Controller 1's latency over 10ms (10.05 ms)
02/10/2021 22:53:31: Controller 1's latency now under 10ms
02/10/2021 22:53:31: Controller 1's latency over 10ms (11.95 ms)
02/10/2021 22:53:31: Controller 1's latency now under 10ms
02/10/2021 22:53:31: Controller 1's latency over 10ms (11.55 ms)
02/10/2021 22:53:32: Controller 3's latency now under 10ms
02/10/2021 22:53:32: Controller 1's latency now under 10ms
02/10/2021 22:53:32: Controller 3's latency over 10ms (10.05 ms)
02/10/2021 22:53:32: Controller 3's latency now under 10ms
02/10/2021 22:53:32: Controller 4's latency now under 10ms
02/10/2021 22:53:32: Controller 2's latency now under 10ms
02/10/2021 22:53:32: Controller 1's latency over 10ms (10 ms)
02/10/2021 22:53:32: Controller 2's latency over 10ms (10 ms)
02/10/2021 22:53:32: Controller 4's latency over 10ms (10.1 ms)
02/10/2021 22:53:32: Controller 3's latency over 10ms (10.05 ms)
02/10/2021 22:53:32: Controller 1's latency now under 10ms
02/10/2021 22:53:32: Controller 4's latency now under 10ms
02/10/2021 22:53:32: Controller 1's latency over 10ms (10.2 ms)
02/10/2021 22:53:32: Controller 4's latency over 10ms (10.25 ms)
02/10/2021 22:53:33: Controller 4's latency now under 10ms
02/10/2021 22:53:33: Controller 4's latency over 10ms (11.75 ms)
02/10/2021 22:53:33: Controller 3's latency now under 10ms
02/10/2021 22:53:33: Controller 3's latency over 10ms (10.05 ms)
02/10/2021 22:53:33: Controller 2's latency now under 10ms
02/10/2021 22:53:33: Controller 2's latency over 10ms (11.6 ms)
02/10/2021 22:53:33: Controller 1's latency now under 10ms
02/10/2021 22:53:33: Controller 1's latency over 10ms (11.85 ms)
02/10/2021 22:53:34: Controller 1's latency now under 10ms
02/10/2021 22:53:34: Controller 4's latency now under 10ms
02/10/2021 22:53:34: Controller 1's latency over 10ms (11.6 ms)
02/10/2021 22:53:34: Controller 4's latency over 10ms (11.4 ms)
02/10/2021 22:53:34: Controller 1's latency now under 10ms
02/10/2021 22:53:34: Controller 1's latency over 10ms (10.05 ms)
02/10/2021 22:53:34: Controller 3's latency now under 10ms
02/10/2021 22:53:34: Controller 3's latency over 10ms (11.6 ms)
02/10/2021 22:53:34: Controller 3's latency now under 10ms
02/10/2021 22:53:34: Controller 3's latency over 10ms (11.75 ms)
02/10/2021 22:53:34: Controller 3's latency now under 10ms
02/10/2021 22:53:34: Controller 2's latency now under 10ms
02/10/2021 22:53:34: Controller 4's latency now under 10ms
02/10/2021 22:53:34: Controller 1's latency now under 10ms
02/10/2021 22:55:23: Controller 1's latency over 10ms (10.35 ms)
02/10/2021 22:55:23: Controller 3's latency over 10ms (11.25 ms)
02/10/2021 22:55:23: Controller 4's latency over 10ms (10.45 ms)
02/10/2021 22:55:23: Controller 2's latency over 10ms (11.2 ms)
02/10/2021 22:55:23: Controller 3's latency now under 10ms
02/10/2021 22:55:23: Controller 1's latency now under 10ms
02/10/2021 22:55:23: Controller 1's latency over 10ms (11.55 ms)
02/10/2021 22:55:23: Controller 3's latency over 10ms (11.65 ms)
02/10/2021 22:55:23: Controller 1's latency now under 10ms
02/10/2021 22:55:23: Controller 1's latency over 10ms (10.15 ms)
02/10/2021 22:55:24: Controller 1's latency now under 10ms
02/10/2021 22:55:24: Controller 1's latency over 10ms (10 ms)
02/10/2021 22:55:24: Controller 4's latency now under 10ms
02/10/2021 22:55:24: Controller 4's latency over 10ms (10 ms)
02/10/2021 22:55:24: Controller 1's latency now under 10ms
02/10/2021 22:55:24: Controller 1's latency over 10ms (11.85 ms)
02/10/2021 22:55:24: Controller 1's latency now under 10ms
02/10/2021 22:55:24: Controller 1's latency over 10ms (10.15 ms)
02/10/2021 22:55:24: Controller 4's latency now under 10ms
02/10/2021 22:55:24: Controller 2's latency now under 10ms
02/10/2021 22:55:24: Controller 1's latency now under 10ms
02/10/2021 22:55:24: Controller 3's latency now under 10ms
02/10/2021 22:55:24: Controller 3's latency over 10ms (10.05 ms)
02/10/2021 22:55:24: Controller 1's latency over 10ms (10 ms)
02/10/2021 22:55:24: Controller 1's latency now under 10ms
02/10/2021 22:55:24: Controller 3's latency now under 10ms
02/10/2021 22:56:29: Controller 1's latency over 10ms (10.95 ms)
02/10/2021 22:56:29: Controller 4's latency over 10ms (10.65 ms)
02/10/2021 22:56:29: Controller 3's latency over 10ms (10.2 ms)
02/10/2021 22:56:29: Controller 2's latency over 10ms (10.1 ms)
02/10/2021 22:56:29: Controller 1's latency now under 10ms
02/10/2021 22:56:29: Controller 1's latency over 10ms (10 ms)
02/10/2021 22:56:29: Controller 1's latency now under 10ms
02/10/2021 22:56:29: Controller 1's latency over 10ms (11.9 ms)
02/10/2021 22:56:29: Controller 1's latency now under 10ms
02/10/2021 22:56:29: Controller 1's latency over 10ms (11.4 ms)
02/10/2021 22:56:30: Controller 1's latency now under 10ms
02/10/2021 22:56:30: Controller 1's latency over 10ms (10.1 ms)
02/10/2021 22:56:30: Controller 1's latency now under 10ms
02/10/2021 22:56:30: Controller 2's latency now under 10ms
02/10/2021 22:56:30: Controller 4's latency now under 10ms
02/10/2021 22:56:30: Controller 3's latency now under 10ms
02/10/2021 22:56:30: Controller 1's latency over 10ms (10.3 ms)
02/10/2021 22:56:30: Controller 2's latency over 10ms (10 ms)
02/10/2021 22:56:30: Controller 2's latency now under 10ms
02/10/2021 22:56:30: Controller 2's latency over 10ms (11.55 ms)
02/10/2021 22:56:30: Controller 4's latency over 10ms (11.45 ms)
02/10/2021 22:56:30: Controller 3's latency over 10ms (11.35 ms)
02/10/2021 22:56:31: Controller 1's latency now under 10ms
02/10/2021 22:56:31: Controller 1's latency over 10ms (10 ms)
02/10/2021 22:56:31: Controller 4's latency now under 10ms
02/10/2021 22:56:31: Controller 4's latency over 10ms (10 ms)
02/10/2021 22:56:31: Controller 3's latency now under 10ms
02/10/2021 22:56:31: Controller 1's latency now under 10ms
02/10/2021 22:56:31: Controller 1's latency over 10ms (11.85 ms)
02/10/2021 22:56:31: Controller 3's latency over 10ms (11.85 ms)
02/10/2021 22:56:31: Controller 2's latency now under 10ms
02/10/2021 22:56:31: Controller 1's latency now under 10ms
02/10/2021 22:56:31: Controller 1's latency over 10ms (10 ms)
02/10/2021 22:56:31: Controller 2's latency over 10ms (10.05 ms)
02/10/2021 22:56:31: Controller 1's latency now under 10ms
02/10/2021 22:56:31: Controller 1's latency over 10ms (11.85 ms)
02/10/2021 22:56:31: Controller 1's latency now under 10ms
02/10/2021 22:56:31: Controller 2's latency now under 10ms
02/10/2021 22:56:31: Controller 3's latency now under 10ms
02/10/2021 22:56:31: Controller 4's latency now under 10ms
02/10/2021 22:56:31: Controller 1's latency over 10ms (10.15 ms)
02/10/2021 22:56:31: Controller 1's latency now under 10ms
02/10/2021 22:56:39: Controller 1's latency over 10ms (10.65 ms)
02/10/2021 22:56:39: Controller 3's latency over 10ms (10.8 ms)
02/10/2021 22:56:39: Controller 4's latency over 10ms (10.25 ms)
02/10/2021 22:56:39: Controller 2's latency over 10ms (11.5 ms)
02/10/2021 22:56:39: Controller 4's latency now under 10ms
02/10/2021 22:56:39: Controller 1's latency now under 10ms
02/10/2021 22:56:39: Controller 1's latency over 10ms (10.45 ms)
02/10/2021 22:56:39: Controller 4's latency over 10ms (10.25 ms)
02/10/2021 22:56:39: Controller 1's latency now under 10ms
02/10/2021 22:56:39: Controller 1's latency over 10ms (10 ms)
02/10/2021 22:56:39: Controller 1's latency now under 10ms
02/10/2021 22:56:39: Controller 1's latency over 10ms (11.75 ms)
02/10/2021 22:56:40: Controller 3's latency now under 10ms
02/10/2021 22:56:40: Controller 1's latency now under 10ms
02/10/2021 22:56:40: Controller 4's latency now under 10ms
02/10/2021 22:56:40: Controller 2's latency now under 10ms
02/10/2021 22:56:40: Controller 2's latency over 10ms (10.1 ms)
02/10/2021 22:56:40: Controller 2's latency now under 10ms
02/10/2021 22:56:40: Controller 2's latency over 10ms (10.3 ms)
02/10/2021 22:56:40: Controller 3's latency over 10ms (10.35 ms)
02/10/2021 22:56:41: Controller 1's latency over 10ms (11.05 ms)
02/10/2021 22:56:41: Controller 4's latency over 10ms (11.8 ms)
02/10/2021 22:56:41: Controller 1's latency now under 10ms
02/10/2021 22:56:41: Controller 1's latency over 10ms (11.6 ms)
02/10/2021 22:56:41: Controller 1's latency now under 10ms
02/10/2021 22:56:41: Controller 1's latency over 10ms (10.05 ms)
02/10/2021 22:56:41: Controller 3's latency now under 10ms
02/10/2021 22:56:41: Controller 2's latency now under 10ms
02/10/2021 22:56:41: Controller 1's latency now under 10ms
02/10/2021 22:56:41: Controller 3's latency over 10ms (11.6 ms)
02/10/2021 22:56:41: Controller 2's latency over 10ms (11.8 ms)
02/10/2021 22:56:41: Controller 1's latency over 10ms (11.75 ms)
02/10/2021 22:56:42: Controller 1's latency now under 10ms
02/10/2021 22:56:42: Controller 1's latency over 10ms (10.2 ms)
02/10/2021 22:56:42: Controller 1's latency now under 10ms
02/10/2021 22:56:42: Controller 1's latency over 10ms (10.35 ms)
02/10/2021 22:56:42: Controller 1's latency now under 10ms
02/10/2021 22:56:42: Controller 4's latency now under 10ms
02/10/2021 22:56:42: Controller 2's latency now under 10ms
02/10/2021 22:56:42: Controller 3's latency now under 10ms
02/10/2021 22:56:42: Controller 4's latency over 10ms (10.25 ms)
02/10/2021 22:56:42: Controller 3's latency over 10ms (10.7 ms)
02/10/2021 22:56:42: Controller 4's latency now under 10ms
02/10/2021 22:56:42: Controller 3's latency now under 10ms
02/10/2021 22:56:42: Controller 3's latency over 10ms (10.05 ms)
02/10/2021 22:56:42: Controller 3's latency now under 10ms
02/10/2021 22:57:50: Controller 2's latency over 10ms (10.45 ms)
02/10/2021 22:57:50: Controller 3's latency over 10ms (10.15 ms)
02/10/2021 22:57:50: Controller 4's latency over 10ms (10.65 ms)
02/10/2021 22:57:50: Controller 3's latency now under 10ms
02/10/2021 22:57:50: Controller 3's latency over 10ms (10.25 ms)
02/10/2021 22:57:50: Controller 1's latency over 10ms (11.75 ms)
02/10/2021 22:57:51: Controller 1's latency now under 10ms
02/10/2021 22:57:51: Controller 1's latency over 10ms (11.7 ms)
02/10/2021 22:57:51: Controller 1's latency now under 10ms
02/10/2021 22:57:51: Controller 1's latency over 10ms (12.1 ms)
02/10/2021 22:57:51: Controller 1's latency now under 10ms
02/10/2021 22:57:51: Controller 1's latency over 10ms (11.65 ms)
02/10/2021 22:57:51: Controller 1's latency now under 10ms
02/10/2021 22:57:51: Controller 3's latency now under 10ms
02/10/2021 22:57:51: Controller 1's latency over 10ms (10.15 ms)
02/10/2021 22:57:51: Controller 3's latency over 10ms (11.95 ms)
02/10/2021 22:57:51: Controller 3's latency now under 10ms
02/10/2021 22:57:51: Controller 1's latency now under 10ms
02/10/2021 22:57:51: Controller 2's latency now under 10ms
02/10/2021 22:57:51: Controller 4's latency now under 10ms
02/10/2021 22:58:03: Controller 2's latency over 10ms (10.15 ms)
02/10/2021 22:58:03: Controller 2's latency now under 10ms
02/10/2021 22:58:03: Controller 2's latency over 10ms (10.25 ms)
02/10/2021 22:58:03: Controller 1's latency over 10ms (11.5 ms)
02/10/2021 22:58:03: Controller 4's latency over 10ms (11.5 ms)
02/10/2021 22:58:03: Controller 3's latency over 10ms (10.6 ms)
02/10/2021 22:58:03: Controller 1's latency now under 10ms
02/10/2021 22:58:03: Controller 1's latency over 10ms (11.6 ms)
02/10/2021 22:58:03: Controller 1's latency now under 10ms
02/10/2021 22:58:03: Controller 3's latency now under 10ms
02/10/2021 22:58:03: Controller 1's latency over 10ms (11.8 ms)
02/10/2021 22:58:03: Controller 3's latency over 10ms (11.95 ms)
02/10/2021 22:58:04: Controller 1's latency now under 10ms
02/10/2021 22:58:04: Controller 3's latency now under 10ms
02/10/2021 22:58:04: Controller 1's latency over 10ms (11.85 ms)
02/10/2021 22:58:04: Controller 3's latency over 10ms (11.7 ms)
02/10/2021 22:58:04: Controller 1's latency now under 10ms
02/10/2021 22:58:04: Controller 2's latency now under 10ms
02/10/2021 22:58:04: Controller 3's latency now under 10ms
02/10/2021 22:58:04: Controller 4's latency now under 10ms
02/10/2021 22:58:04: Controller 4's latency over 10ms (10.2 ms)
02/10/2021 22:58:04: Controller 3's latency over 10ms (10.25 ms)
02/10/2021 22:58:04: Controller 3's latency now under 10ms
02/10/2021 22:58:04: Controller 4's latency now under 10ms
02/10/2021 22:58:24: Controller 2's latency over 10ms (10.95 ms)
02/10/2021 22:58:24: Controller 4's latency over 10ms (11.1 ms)
02/10/2021 22:58:24: Controller 3's latency over 10ms (11.1 ms)
02/10/2021 22:58:24: Controller 1's latency over 10ms (10.9 ms)
02/10/2021 22:58:24: Controller 2's latency now under 10ms
02/10/2021 22:58:24: Controller 2's latency over 10ms (11.75 ms)
02/10/2021 22:58:24: Controller 2's latency now under 10ms
02/10/2021 22:58:24: Controller 2's latency over 10ms (11.75 ms)
02/10/2021 22:58:24: Controller 1's latency now under 10ms
02/10/2021 22:58:24: Controller 1's latency over 10ms (10.2 ms)
02/10/2021 22:58:25: Controller 1's latency now under 10ms
02/10/2021 22:58:25: Controller 1's latency over 10ms (10.05 ms)
02/10/2021 22:58:25: Controller 1's latency now under 10ms
02/10/2021 22:58:25: Controller 1's latency over 10ms (11.85 ms)
02/10/2021 22:58:25: Controller 1's latency now under 10ms
02/10/2021 22:58:25: Controller 1's latency over 10ms (10.2 ms)
02/10/2021 22:58:25: Controller 2's latency now under 10ms
02/10/2021 22:58:25: Controller 1's latency now under 10ms
02/10/2021 22:58:25: Controller 3's latency now under 10ms
02/10/2021 22:58:25: Controller 4's latency now under 10ms
02/10/2021 22:58:25: Controller 4's latency over 10ms (10.25 ms)
02/10/2021 22:58:25: Controller 3's latency over 10ms (10.2 ms)
02/10/2021 22:58:25: Controller 4's latency now under 10ms
02/10/2021 22:58:25: Controller 3's latency now under 10ms
02/10/2021 22:58:56: Gamepad data write connection is lost. Disconnecting the gamepad. LastErrorCode=1167
02/10/2021 22:58:56: Gamepad data write connection is lost. Disconnecting the gamepad. LastErrorCode=1167
02/10/2021 22:58:56: Gamepad data write connection is lost. Disconnecting the gamepad. LastErrorCode=1167
02/10/2021 22:58:56: Disassociate X360 Controller for input DS4 #1
02/10/2021 22:58:56: Disassociate X360 Controller for input DS4 #4
02/10/2021 22:58:56: Disassociate X360 Controller for input DS4 #2
02/10/2021 22:58:56: Unplugging virtual X360 Controller
02/10/2021 22:58:56: Unplugging virtual X360 Controller
02/10/2021 22:58:56: Unplugging virtual X360 Controller
02/10/2021 22:58:56: Controller 1 was removed or lost connection
02/10/2021 22:58:56: Controller 2 was removed or lost connection
02/10/2021 22:58:56: Controller 4 was removed or lost connection
02/10/2021 22:58:56: Gamepad data write connection is lost. Disconnecting the gamepad. LastErrorCode=1167
02/10/2021 22:58:56: Disassociate X360 Controller for input DS4 #3
02/10/2021 22:58:56: Unplugging virtual X360 Controller
02/10/2021 22:58:56: Controller 3 was removed or lost connection
```

Just bought 3 more controllers today and am laying with 4 controllers that don't work (and I don't even have a PS4 lmao)

Additional thing I tried: Disabling "Enable Output data to DS4". Doesn't solve it.

Hope someone can help me :)

**Comment by Ryochan7** [comment]
It might just be your BT adapter. When I used an old BT 2.1+ adapter, latency would jump up big time when two controllers were connected. With my current BT 4.0 adapter, it can handle 4 controllers being connected at once with no latency issues. Although, the adapter outright refuses to connect a fifth controller but maybe that is a Windows BT limitation; I don't know.

Testing beyond 4 controllers requires some being plugged in via USB. I have tested 5 controllers (4 BT, 1 USB) opened in DS4Windows at one time and it worked fine on my system.

**Comment by Fusseldieb** [comment]
Thanks for the quick reply :D

Well, it's a newer ROG laptop from 2019, so it really shouldn't be the bluetooth adapter, since it's probably already Bluetooth 4.X or so, but well...

Since you said that yours work fine, I'm a little bit relieved.

I just purchased a cheap bluetooth dongle with "V5.0" written on it. I hope that'll work. I also saw some written "CSR", but reading the README told me that they most likely wouldn't work, so I am *trying* to avoid them.  I hope that "V5.0" is not CSR then.
